# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/plasma-login-manager/bin-x11-x.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

plasmalogin-helper-start-x11user[25800]: Failed to start "/usr/bin/X -nolisten tcp -background none -seat seat0 ...

--- plasma-login-manager/src/common/Configuration.h.orig	2026-01-21 15:43:22.141688094 +0100
+++ plasma-login-manager/src/common/Configuration.h	2026-01-21 15:43:59.735261071 +0100
@@ -28,7 +28,7 @@
         Entry(Namespaces,          QStringList, QStringList(),                                  _S("Comma-separated list of Linux namespaces for user session to enter"));
         // TODO: Not absolutely sure if everything belongs here. Xsessions, VT and probably some more seem universal
         Section(X11,
-            Entry(ServerPath,          QString,     _S("/usr/bin/X"),                           _S("Path to X server binary"));
+            Entry(ServerPath,          QString,     _S("/usr/X11/bin/X"),                           _S("Path to X server binary"));
             Entry(ServerArguments,     QString,     _S("-nolisten tcp"),                        _S("Arguments passed to the X server invocation"));
             Entry(SessionLogFile,      QString,     _S(".local/share/plasmalogin/xorg-session.log"),   _S("Path to the user session log file"));
         );
